### ISSUE TITLE
Hosting of non-owned static javascript

### ISSUE THREAD

**Issue by kurtlewis** [zacharysang/flock#11]
We need to host two javascript files:
`/easyrtc/easyrtc.js`
`/socket.io/socket.io.js`

We need to figure out how we want to manage this.

1. We could rehost the javascript files not in the repo (I don't know where they are right now?)
2. We could copy the files into our repo (would need to check the license on redistribution)
2. We could link to an existing source of the files online

Any other options?

**Comment by zacharysang** [comment]
Yeah, we could do that. If we did that though, are you thinking about moving our own libraries (flock.js and flock-mpi.js) into that webserver as well? I think these should all be served from the same webserver since they are dependencies for each other.

**Comment by zacharysang** [comment]
Sounds good to me

**Comment by zacharysang** [comment]
* [socket.io.js](https://demo.easyrtc.com/socket.io/socket.io.js)
* [easyrtc.js](https://demo.easyrtc.com/easyrtc/easyrtc.js)
